invoice/src/main/java/org/killbill/billing/invoice/template/formatters/DefaultInvoiceFormatter.java

```diff
@@ -71,6 +72,18 @@
     private final InternalTenantContext context;
     private final ResourceBundleFactory bundleFactory;
 
+    public static Map<java.util.Currency, Locale> currencyLocaleMap;
```

Comment:
Could you put this initialization in the constructor of `DefaultInvoiceFormatterFactory` (and change the `DefaultInvoiceFormatter` constructor to accept it as an argument)? `static` initialization makes me nervous...

Also, feel free to expose a package protect getter in `DefaultInvoiceFormatterFactory`, so it's easier to update tests (e.g. `TestDefaultInvoiceFormatter`).